instrumentation/apache-httpclient/apache-httpclient-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v2_0/CommonsHttpClientTracer.java

```diff
@@ -21,6 +22,10 @@ public static CommonsHttpClientTracer tracer() {
     return TRACER;
   }
 
+  public Context startSpan(Context parentContext, HttpMethod request) {
+    return super.startSpan(parentContext, request, request);
+  }
```

Comment:
The goal is to make the super.startSpan() `protected`, and only expose simplified startSpan to users of the Tracer